filament/src/driver/Program.cpp

```diff
@@ -64,12 +53,15 @@ Program& Program::addUniformBlock(size_t index, const UniformInterfaceBlock* ib)
     return *this;
 }
 
-Program& Program::addSamplerBlock(size_t index, const SamplerInterfaceBlock* sb) {
-    mSamplerInterfaceBlocks[index] = sb;
-    mSamplerCount++;
+Program& Program::addSamplerGroup(size_t bindingPoint, const Program::Sampler* samplers, size_t count) {
```

Comment:
Maybe `setSamplerGroup` is a better name since it overwrites the sampler group at this binding point.